bookkeeper-server/src/main/java/org/apache/bookkeeper/proto/BookieServer.java

```diff
@@ -159,6 +162,28 @@ public synchronized void shutdown() {
         running = false;
     }
 
+    /**
+     * Ensure the current user can start-up the process if it's restricted.
+     */
+    private void validateUser(ServerConfiguration conf) throws AccessControlException {
+        if (conf.containsKey("permittedStartupUsers")) {
```

Comment:
use the constant defined in AbstractConfiguration.

Reply:
Will do, thanks.